packages/st2/rpm/st2.spec

```diff
@@ -14,9 +14,20 @@ Epoch: %{epoch}
 %if 0%{?use_st2python}
 Requires: st2python, python-devel, openssl-devel, libffi-devel, git, pam, openssh-server, openssh-clients, bash, setup
 %else
-Requires: python-devel, openssl-devel, libffi-devel, git, pam, openssh-server, openssh-clients, bash, setup
+Requires: python3-devel openssl-devel, libffi-devel, git, pam, openssh-server, openssh-clients, bash, setup
```

Comment:
Wouldn't this apply to `el7` as well? Need to make it as a requirement for `el8` only.

Reply:
I can see your point. I'll update it.